src/Components/Web/src/Web/RenderTreeBuilderExtensions.cs

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Components.Rendering;
+using Microsoft.AspNetCore.Components.RenderTree;
+
+namespace Microsoft.AspNetCore.Components.Web
+{
+    /// <summary>
+    /// Provides methods for building a collection of <see cref="RenderTreeFrame"/> entries.
+    /// </summary>
+    public static class RenderTreeBuilderExtensions
```

Comment:
We should give this a different class name like `WebRenderTreeBuilderExtensions` or else it will be hard to search. We try to do this when there's a fan-out of libraries that will all *extend* a common API.

Reply:
Done